### PR TITLE
Phar GitHub actions

### PR DESCRIPTION
**Pull request by Jeckerson**
Working example from fork - https://github.com/Jeckerson/phpbench-1/runs/1468396755?check_suite_focus=true

image

### CODE REVIEW

**Comment by dantleech** [comment]
Awesome, thankyou :)

File: .github/workflows/release-phar.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          composer install
+          wget https://github.com/box-project/box/releases/download/3.9.1/box.phar
+          php box.phar compile -c box.json.gh-release
+      - name: Check existence of compiled .phar
+        run: test -e phpbench.phar && exit 0 || exit 10
+
+      - name: Upload to Release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (dantleech) [inline .github/workflows/release-phar.yml:27]: assume this will "just work" ?

Comment (Jeckerson) [reply .github/workflows/release-phar.yml:27]: Yes

**Comment by Jeckerson** [comment]
Also, do not forget to remove related logic of travis CI.

**Pull request merged**